Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm

```diff
@@ -160,6 +160,8 @@ static void launchWithExtensionKit(ProcessLauncher& processLauncher, ProcessLaun
         if (error) {
             NSLog(@"Error launching process %@ error %@", process, error);
             RELEASE_LOG_FAULT(Process, "Error launching process, descriptrion '%s', reason '%s'", String([error localizedDescription]).utf8().data(), String([error localizedFailureReason]).utf8().data());
+#if !PLATFORM(IOS)
```

Comment:
But here, you will launch XPC for IOS_SIMULATOR. One of these isn't right, I think?

Reply:
Ah, that's a good point! Fixed in latest patch.

Thanks for reviewing!

Reply:
Wait, so the plan is to only use extensions on iOS devices? Why?

Reply:
The plan is to use it on all platforms, this will be removed once we enable it on the remaining platforms.

Thanks for reviewing!

Reply:
Yes, this is iOS-only, but we want to avoid building/launching the XPC service on simulators _and_ devices, right?

If so: `PLATFORM(IOS)` is correct (it's true for simulators and devices), and the xcconfig filters should be `sdk=iphone*x.y` like you originally had (to match `iphoneos` and `iphonesimulator` SDK names).

> But here, you will launch XPC for IOS_SIMULATOR.

Basically, I think this is wrong. PLATFORM(IOS) is `TARGET_OS_IOS`, which true for device and simulator variants.